.github/actions/build-rpms/main.js

```diff
@@ -0,0 +1,35 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+"use strict";
+const child_process = require("child_process");
+const spawnArgs = {
+	stdio: "inherit",
+	stderr: "inherit",
+};
+
+let atcComponent = process.env.ATC_COMPONENT;
```

Comment:
Since the change is no longer conditional, this can just be:
```javascript
const atcComponent = `${process.env.ATC_COMPONENT}_build`;

Reply:
That would complicate validation for `process.env.ATC_COMPONENT`. The condition for the check that exits if `ATC_COMPONENT` is undefined or empty would become `atcComponent === "undefined_build"`, which is bad.